packages/experimental/Shimmer/src/ShimmerTokens.android.ts

```diff
@@ -1 +1,15 @@
-export { defaultShimmerTokens } from './ShimmerTokens.mobile';
+import type { Theme } from '@fluentui-react-native/framework';
+import type { TokenSettings } from '@fluentui-react-native/use-styling';
+
+import type { ShimmerTokens } from './Shimmer.types';
+
+export const defaultShimmerTokens: TokenSettings<ShimmerTokens, Theme> = (theme: Theme) =>
+  ({
+    angle: 45,
```

Comment:
Curious why this value differs?

Reply:
Initially this value was 0, it became 45 when common token values were shared with android, reverting back to original based on design input